client/me/notification-settings/reader-subscriptions/index.jsx

```diff
@@ -25,270 +25,259 @@ import { recordGoogleEvent } from 'calypso/state/analytics/actions';
 import { getReaderTeams } from 'calypso/state/teams/selectors';
 import SubscriptionManagementBackButton from '../subscription-management-back-button';
 
-class NotificationSubscriptions extends Component {
-	handleClickEvent( action ) {
-		return () => this.props.recordGoogleEvent( 'Me', 'Clicked on ' + action );
-	}
+const NotificationSubscriptions = ( props ) => {
+	const dispatch = useDispatch();
+	const teams = useSelector( getReaderTeams );
+	const isAutomattician = isAutomatticTeamMember( teams );
+	const { translate } = props;
```

Comment:
I see we're referencing props a lot with `props.something` in this PR. How about we destructure them all right in the function definition argument, and we'll be able to shave off some of that.